docs/csharp/language-reference/operators/null-coalescing-operator.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: "?? operator - C# reference"
+title: "?? and ??= operators - C# reference"
 ms.custom: seodec18
-ms.date: 06/07/2019
+ms.date: 09/09/2019
 f1_keywords: 
   - "??_CSharpKeyword"
 helpviewer_keywords: 
   - "null-coalescing operator [C#]"
   - "?? operator [C#]"
 ms.assetid: 088b1f0d-c1af-4fe1-b4b8-196fd5ea9132
 ---
-# ?? operator (C# reference)
+# ?? and ??= operators (C# reference)
 
 The null-coalescing operator `??` returns the value of its left-hand operand if it isn't `null`; otherwise, it evaluates the right-hand operand and returns its result. The `??` operator doesn't evaluate its right-hand operand if the left-hand operand evaluates to non-null.
```

Comment:
This paragraph is under `# ?? and ??= operators (C# reference)` heading, but it also talks about `??` but not `??=`

There is a `## Null-coalescing assignment` that talks about `??=`, so I suggest adding `## Null-coalescing operator` just under `# ?? and ??= operators (C# reference)`.

Reply:
For the reference, I prefer the single H2, as this article has now.

I agree that in the standard, we would have separate headings for each sub clause.

Reply:
@Youssef1313 thanks for your remark. It contributed to the final version of the article.